utils/common/src/main/kotlin/ArchiveUtils.kt

```diff
@@ -163,7 +167,7 @@ fun File.unpackDeb(targetDirectory: File, filter: (ArchiveEntry) -> Boolean = {
             subDirectory.safeMkdirs()
 
             val file = tempDir.resolve(name)
-            file.unpack(subDirectory, filter)
+            file.unpack(subDirectory, filter = filter)
```

Comment:
Unnecessary change.

Reply:
This is actually needed because of the new optional parameter in `unpack()`.